### DISCUSSION TITLE
Unclear instructions

### DISCUSSION THREAD

**Discussion by charminULTRA** [dz0ny/klipper-prusa-mk3s#52] [General]
Just quick note that the instructions in the read me could use a lot of work. I'd encourage you to add more detail.

1) Why do I need to use serial rather than USB?
2) Where am I entering the terminal commands?
3) How do I access the config?
4) Why does the building and flashing microcontroller instruction link assume we'll be using OctoPrint?

These are all not answered, but critical missing items. I figured it out on my own, but these are basic steps that should be included.

Answers:
1) No reason
2) Inside of the Mainsail web interface, which is located at the IP address of your Pi
3) Inside of the Mainsail web interface, under the config folder.
4) Unknown

**Comment by confuzled-dev** [comment]
TBH, I think assuming someone is going to use Octoprint with a klipperized MK3+ in today's landscape is incorrect anyways. Instead of leading someone to a suboptimal octoprint experience simply because it is what they are likely used to, explain the benefits of using something like fluidd or mainsail and assist in bridging that gap. 

Once they are on klipper, not using one of these two environments is going to be a barrier to getting assistance when things go wrong. 
I think it is important to also reference both the klipper and prusa3d discord in these documents. That being said, most prusa users use prusa printers because they value consistency over speed or feature set, however, I have not fould this to be a meaningful barrier to getting assistance. 

I will get my config files hosted on github and post them for you to review. Maybe some of the changes I have made along the way make sense for a initial starting point, or maybe they will address things that while possibly not set correctly as a baseline, arent actually addressed in this repo. 

One MAJOR barrier to gettign assistance since i started has been all the nested config files. When I have had issues the common theme is always "why the hell do you have so many cfg files, this is hard to follow" I think flattening things a bit might be helpful. I am sure it helps from a developmental standpoint to have things isolated to work on but the overwhelming majority of klipper users do not structure they setups this way. 

@charminULTRA have you forked this repo?

**Comment by skelly4900** [comment]
Does a Pi Zero 2W or better have to be used, or can we use the lower performance original Pi Zero W?

**Comment by confuzled-dev** [comment]
You will not have a good time with a pi zero.

Check out the GitHub of the person above you and check it a project being
worked on to modernize the process of putting klipper on a prusa
